jib-gradle-plugin/README.md

```diff
@@ -175,7 +175,8 @@ Field | Type | Default | Description
 `to` | [`to`](#to-closure) | *Required* | Configures the target image to build your application to.
 `from` | [`from`](#from-closure) | See [`from`](#from-closure) | Configures the base image to build your application on top of.
 `container` | [`container`](#container-closure) | See [`container`](#container-closure) | Configures the container that is run from your built image.
-`extraDirectory` | [`extraDirectory`](#extradirectory-closure) / `File` | `(project-dir)/src/main/jib` | Configures the directory used to add arbitrary files to the image.
+`extraDirectory` | `extraDirectory` / `File` | `(project-dir)/src/main/jib` | Deprecated. Use `extraDirectories`.
```

Comment:
I think we can just remove this, we don't want people to use it so it just takes up space.